bam_import.c

```diff
@@ -251,6 +251,9 @@ static int import_fastq(int argc, char **argv, opts_t *opts) {
     } else {
         hdr_out = sam_hdr_init();
     }
+    
+    // Update sort order of output header
+    hdr_out = sam_hdr_update_line(hdr_out, "HD", "SO", "unsorted", "GO", "query", NULL);
```

Comment:
`sam_hdr_update_line()` returns an integer [indicating success or failure](https://github.com/samtools/htslib/blob/e51f72f0/htslib/sam.h#L579), not a new header.